NEWS.md

```diff
@@ -1,4 +1,11 @@
-# tinkr dev
+# tinkr 0.0.0.9002
```

Comment:
You're so good at versioning that you might enjoy using fledge? https://cynkra.github.io/fledge/